packages/react-data-grid-examples/src/scripts/example03-frozen-cols.js

```diff
@@ -82,12 +88,12 @@ class Example extends React.Component {
   }
 }
 
-const exampleDescription = <p>To make a given column frozen, set <code>column.locked = true</code>. In this example, the ID columns has been frozen and will remain in position as you scroll horizontally</p>
+const exampleDescription = <p>To make a given column frozen, set <code>column.frozen = true</code>. In this example, the ID columns has been frozen and will remain in position as you scroll horizontally</p>
```

Comment:
We should change the example to use `frozen` instead of `locked` to avoid any confusion